Examples/Algorithms/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ add_subdirectory_if(HepMC ACTS_BUILD_EXAMPLES_HEPMC3)
 add_subdirectory(Fatras)
 add_subdirectory_if(Geant4HepMC ACTS_BUILD_EXAMPLES_GEANT4 AND ACTS_BUILD_EXAMPLES_HEPMC3)
 add_subdirectory_if(Geant4 ACTS_BUILD_EXAMPLES_GEANT4)
-add_subdirectory_if(Geant4DD4hep ACTS_BUILD_EXAMPLES_DD4HEP AND ACTS_BUILD_EXAMPLES_GEANT4)
+# add_subdirectory_if(Geant4DD4hep ACTS_BUILD_EXAMPLES_DD4HEP AND ACTS_BUILD_EXAMPLES_GEANT4)
```

Comment:
Obsolete? If so can we remove this line?

Reply:
Removed.